Assets/LeapMotion/Core/Scripts/LeapXRServiceProvider.cs

```diff
@@ -217,9 +213,19 @@ public enum TemporalWarpingMode {
       }
       #endif
 
-      var effTransformPose = new Pose(InputTracking.GetLocalPosition(XRNode.CenterEye),
-                                      InputTracking.GetLocalRotation(XRNode.CenterEye))
-                               .Then(_trackingBaseDeltaPose);
+      // Get most recent tracked pose.
+      var trackedPose = new Pose(InputTracking.GetLocalPosition(XRNode.CenterEye),
+                                 InputTracking.GetLocalRotation(XRNode.CenterEye));
+
+      // If we don't know of any pose offset yet, account for it by finding the pose
+      // delta from the "local" tracked pose to the actual camera pose.
+      if (!_trackingBaseDeltaPose.HasValue) {
+        _trackingBaseDeltaPose = Camera.main.transform.ToWorldPose().From(trackedPose);
```

Comment:
We shouldn't use Camera.main, it may not even exist in the scene, or be the actual camera we are attached to.  I think _cachedCamera is the one to use?

Reply:
Ah sure, good catch